### PR TITLE
Add Regulations page and Organizer guidelines

### LINKED ISSUE #1593: Add a WRC section
It's difficult to find the relevant issue in the WRC repository because people assume it's here. Therefore, I am adding this here so things won't get lost.

The WRC would like to have a section on the website sooner or later.
I created a draft [here](https://github.com/thewca/wca-regulations/issues/419#issuecomment-294306580).
Noobish implementation is [here](https://github.com/Laura-O/worldcubeassociation.org/tree/WRC-section).

The news section could (or should) be implemented together with other plans (especially the plans the WCT has or might come up with).

### PR DESCRIPTION
**Pull request by Laura-O**
Fixes #1593 

* Page about regulations with links/explanations
* Static page with organizer guidelines
* Link to the "About the Regulations"-page in the dropdown menu
* Icons changed (guidelines are not a question mark anymore...)

Screenshots:
[About page](http://imgur.com/qvGhtqe)
[Organizer Guidelines](http://imgur.com/DHjQmqk)

I decided to make both the regulations page and the organizer guidelines translatable. That gives translators a lot of additional work, but I think that's worth it. :-)

### CODE REVIEW

File: WcaOnRails/config/locales/en.yml [hunk 1]
```diff
@@ -1302,3 +1303,79 @@ en:
...
+    paragraph3:
+      title: "Deadlines for the announcement of the competition"
+      content: "The competition should be announced at least one month before the start of the competition (see Guideline 8a4++). If there are special reasons preventing this deadline to be met, competitions may be announced up to two weeks before, at the discretion
+      of the WCA Board. It is highly recommended to inform the WCA Board of these reasons as soon as possible to avoid a later refusal of the competition."
+    paragraph4:
+      title: "Proximity policy"
+      content1: "As a general policy, WCA Competitions will be accepted if they are at least 200 km or 26 days away from any other competition. WCA Board should particularly review a proposed competition if there is another competition for which the distance between
+      them is less than 200 km and the time between them is less than 27 days. A delegate requesting approval for such a competition must provide further information on why the competition should still be accepted. Possible arguments that will support such
```
Comment (jfly) [inline WcaOnRails/config/locales/en.yml]: To me, "at least 200km or 26 days" means `>= 200 km or >= 26 days`, and the inverse of that is `< 200 km and < 26 days`.

File: WcaOnRails/config/routes.rb [hunk 2]
```diff
@@ -108,6 +108,7 @@
   get 'logo' => 'static_pages#logo'
   get 'wca-workbook-assistant' => 'static_pages#wca_workbook_assistant'
   get 'wca-workbook-assistant-versions' => 'static_pages#wca_workbook_assistant_versions'
+  get 'organizer_guidelines' => 'static_pages#organizer_guidelines'
```
Comment (jfly) [inline WcaOnRails/config/routes.rb]: I personally prefer dashes to underscores in urls, so `organizer-guidelines` instead.

File: WcaOnRails/config/i18n-tasks.yml.erb [hunk 3]
```diff
@@ -67,7 +67,9 @@ search:
     - app/assets/images
     - app/assets/fonts
     # i18n-tasks fails on TNoodle jars and old non-utf8 translations...
-    - app/views/regulations
+    - app/views/regulations/history
```
Comment (jfly) [inline WcaOnRails/config/i18n-tasks.yml.erb]: Do you remember what this change was for?

File: WcaOnRails/app/views/static_pages/organizer_guidelines.erb [hunk 4]
```diff
@@ -0,0 +1,55 @@
+<% provide(:title, 'WCA Organizer Guidelines') %>
```
Comment (jfly) [inline WcaOnRails/app/views/static_pages/organizer_guidelines.erb]: Is this intentionally not translated?

File: WcaOnRails/app/views/regulations/about/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,43 @@
+<% provide(:title, 'About the WCA Regulations') %>
```
Comment (jfly) [inline WcaOnRails/app/views/regulations/about/index.html.erb]: Is this intentionally not translated?

**Comment by jonatanklosko** [comment]
@jfly I guess you meant the `#original_hash:` comment, then I think you're right. Another proper and not that hard way would be to ask Philippe to translate this single key and generate sha1 digit manually.

File: WcaOnRails/config/i18n-tasks.yml.erb (hunk 3 above)
Comment (Laura-O) [inline WcaOnRails/config/i18n-tasks.yml.erb]: Yep, `app/views/regulations` was excluded from the translations. I added `app/views/regulations/about`, so I had to change this here to make the translations work for the about page.

File: WcaOnRails/app/views/static_pages/organizer_guidelines.erb (hunk 4 above)
Comment (Laura-O) [inline WcaOnRails/app/views/static_pages/organizer_guidelines.erb]: No, I just forgot to include the titles.

File: WcaOnRails/app/views/regulations/about/index.html.erb (hunk 5 above)
Comment (Laura-O) [inline WcaOnRails/app/views/regulations/about/index.html.erb]: Same for this title.

**Comment by Laura-O** [comment]
Thanks for the feedback, I nearly forgot about this PR...

The content was checked by WRC members, so I hope it's fine. I will update the logic error when I am updating the other mentioned things.

I tried to make a screenshot of the navbar, but that didn't work because the dropdown menu automatically closed when the screenshot was taken. I also used another tool with delay option and a different browser, but still the same behavior. Maybe that's a problem with the WM, I don't know. :-/
But I will take this challenge and give it a try on a different machine. ;-)

**Comment by jfly** [comment]
Oh, one trick for capturing screenshots like that in chrome: open the developer tools, and then when you press F8, it will cause the page to "pause", which should allow you to take screenshots at your leisure.

**Comment by Laura-O** [comment]
Ah, neat trick, thanks. :-)
Here's the screenshot of the dropdown menu:

navbar

**Comment by jfly** [comment]
Thanks for the screenshot.

When can we get this merged up? It looks like my previous comments are still unaddressed?

**Comment by viroulep** [comment]
> @jfly I guess you meant the #original_hash: comment, then I think you're right. Another proper and not that hard way would be to ask Philippe to translate this single key and generate sha1 digit manually.

Just in case it's needed: translation for "About the Regulations" would be "À propos du Règlement", and the original hash would be "6bce1b5".

**Comment by Laura-O** [comment]
> When can we get this merged up? It looks like my previous comments are still unaddressed?

Yes, sorry for the delay. I will try to update the PR this weekend.

**Comment by jfly** [comment]
Awesome, tests pass now! @Laura-O, this LGTM. Again, I have not read the actual content closely, although I'm sure you'll get a bunch of eyes on it when our translators jump in =)

@Laura-O, please reach out to me (or any member of the WST) when you'd like this PR to be merged up and deployed.

**Comment by Laura-O** [comment]
I changed the path of the organizer guidelines, but didn't change the link.. :-(
But that's fixed now and it's ready to be merged.

**Comment by pedrosino** [comment]
I'm not a native english speaker, but this one caught my attention when translating:

> The WCA Regulations **are not only** available in English, **but** WCA members have translated them into several languages: <a href="/regulations/translations/">Translations</a>.

The two sentences don't seem to match well. I think the "but" should be replaced with something else, like "since" or "because".

**Pull request merged**